Regression/WarpX-tests.ini

```diff
@@ -730,53 +730,53 @@ doVis = 0
 compareParticles = 0
 analysisRoutine =  Examples/Tests/radiation_reaction/test_const_B_analytical/analysis_classicalRR.py
 
```

Comment:
@WeiqunZhang This will also deactivate the QED tests on Travis CI. 
I am not sure that this is desirable: the QED tests do compile and pass on Travis CI.

Is there a way to run these tests only on Travis CI, but not on battra?

Reply:
I don't think there is an easy way.  Maybe we should try to make it work on battra.  It fails to compile because

```./Source/QED/QuantumSyncEngineInnards.H:10:10: fatal error: quantum_sync_engine_ctrl.h: No such file or directory
 #include <quantum_sync_engine_ctrl.h>
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Where is `quantum_sync_engine_ctrl.h`?  @lucafedeli88

Reply:
Oh sorry, somehow I missed this conversation... PICSAR should be on the QED branch to compile with QED effects enabled. `quantum_sync_engine_ctrl.h` exists only in this branch. This is because I am introducing some small changes in the PICSAR-QED library in order to better fit the needs of WarpX and I am pushing them to the QED branch (at some point they will be merged into master). Indeed, `run_test.sh` contains the following lines: 
```
if [ ${HAS_QED} = "TRUE" ]; then
    git clone --branch QED https://bitbucket.org/berkeleylab/picsar.git
else
    git clone --branch master https://bitbucket.org/berkeleylab/picsar.git
fi
```
Is this an issue for battra?